### PR TITLE
private/pedro/closebutton fix size margin add separator

### PR DESCRIPTION
**Pull request by pedropintosilva**
- **Close button: Fix tooltip**
- **Close button: introduce separator and fix size and margin**

### CODE REVIEW

File: browser/src/control/Control.UIManager.js [hunk 1]
```diff
@@ -790,6 +790,7 @@ L.Control.UIManager = L.Control.extend({
 		$('.main-nav').hide();
 		if (L.Params.closeButtonEnabled) {
 			$('#closebuttonwrapper').hide();
+			$('#closebuttonwrapperseparator').hide();
```
Comment (eszkadev) [inline browser/src/control/Control.UIManager.js:793]: you forgot to add also "show" in above function showMenubar

File: browser/src/control/Control.Toolbar.js [hunk 2]
```diff
@@ -1067,10 +1067,11 @@ function setupToolbar(e) {
...
 		$('#closebuttonwrapper').css('display', 'block');
-		$('#closebuttonwrapper').prop('title', _('Close document'));
-		map.uiManager.enableTooltip($('#closebuttonwrapper'));
+		$('#closebutton').prop('title', _('Close document'));
+		map.uiManager.enableTooltip($('#closebutton'));
 	} else if (!L.Params.closeButtonEnabled) {
 		$('#closebuttonwrapper').hide();
+		$('#closebuttonwrapperseparator').hide();
 	} else if (L.Params.closeButtonEnabled && !window.mode.isMobile()) {
 		$('#closebuttonwrapper').css('display', 'block');
```
Comment (eszkadev) [inline browser/src/control/Control.Toolbar.js:1076]: don;t we need display block here too?
possibly the default should be hidden state so we will not show it on mobile... have to check

File: browser/src/control/Control.UIManager.js (hunk 1 above)
Comment (eszkadev) [reply browser/src/control/Control.UIManager.js:793]: it works here on mode switch, but it is needed to make this functions work properly if used outside of that UI switch (eg. postmessage) I think

Comment (pedropintosilva) [reply browser/src/control/Control.UIManager.js:793]: good catch, thanks!

File: browser/src/control/Control.UIManager.js [hunk 3]
```diff
@@ -767,6 +767,7 @@ L.Control.UIManager = L.Control.extend({
 		$('.main-nav').show();
 		if (L.Params.closeButtonEnabled && !window.mode.isTablet()) {
 			$('#closebuttonwrapper').show();
+			$('#closebuttonwrapperseparator').show();
```
Comment (pedropintosilva) [inline browser/src/control/Control.UIManager.js:770]: added `$('#closebuttonwrapperseparator').show();` to `showMenubar()` function

File: browser/src/control/Control.UIManager.js (hunk 1 above)
Comment (pedropintosilva) [reply browser/src/control/Control.UIManager.js:794]: added `$('#closebuttonwrapperseparator').show();` to `showMenubar()` function

**Review by eszkadev — APPROVED** [review]
looks good

**Pull request merged**